docs/en/manuals/libraries.md

```diff
@@ -103,6 +103,24 @@ Now the folders that you shared appear in the *Assets pane* and you can use ever
 
 ![Library setup done](images/libraries/libraries_done.png)
 
+## Editing Files in Library Dependencies
+
+Files in libraries cannot be saved.
+You can make changes, and the editor will be able to build with those changes, which is useful for testing.
+However, the file itself remains unchanged, and all modifications will be discarded when the file is closed.
```

Comment:
I prefer if we don't add linebreaks in the markdown. I know they will not have any effect in the generated html and lines 108-110 will still be rendered in a single html paragraph, but from a stylistic perspective we don't add linebreaks within text that will be rendered as a single paragraph anywhere else in the documentation.

Reply:
Thank you, fixed

Reply:
If it was the only issue, let's merge it?